keyboards/canary/canary60rgb/info.json

```diff
@@ -0,0 +1,80 @@
+{
+    "keyboard_name": "CANARY60RGB",
+    "url": "",
+    "maintainer": "canary",
+    "width": 15,
+    "height": 5,
```

Comment:
```suggestion
```

Reply:
https://beta.docs.qmk.fm/developing-qmk/qmk-reference/reference_info_json
Based on the link above, info.json should have "width" and "height" parameters.

Reply:
https://github.com/qmk/qmk_firmware/pull/14274